### PR TITLE
Html template fin

### PR DESCRIPTION
**Pull request by MiTo0o**
wanna merge, so I can open anther branch to work on the post req

### CODE REVIEW

File: command-line-local-shortener/templates/index.tmpl [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+<head>
+    <title>Golang URL shortener</title>
+</head>
+
+<body>
+    <p>example paragraph<strong>example</strong> example
+        <strong>example</strong>example</p>
+
+    <form action="http://localhost:8090" method="POST">
```
Comment (BradHacker) [inline command-line-local-shortener/templates/index.tmpl]: Should probably be something like `action="/create"`. The browser automatically assumes it's using the same domain name (ie. `http://localhost:8090`).

File: command-line-local-shortener/templates/index.tmpl [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+        <label for="fname">Custom ID (auto generates if left blank):</label><br>
+        <input type="text" id="shortID" name="shortID"><br>
```
Comment (BradHacker) [inline command-line-local-shortener/templates/index.tmpl]: Make field `readonly` for now. We will fix how things display later.

File: command-line-local-shortener/templates/index.tmpl [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+        <label for="lname">URL:</label><br>
+        <input type="text" id="longURL" name="longURL">
+        <button>Send my greetings</button>
```
Comment (BradHacker) [inline command-line-local-shortener/templates/index.tmpl]: Button needs `type="submit"`

File: command-line-local-shortener/main.go [hunk 4]
```diff
@@ -107,10 +107,12 @@ func main() {
...
 	router := gin.Default()
 
+	router.LoadHTMLGlob("templates/*")
 	router.GET("/:id", getReq)
-
-	router.GET("/home", func(c *gin.Context) {
-		c.JSON(200, gin.H{"hello": "world"})
+	router.GET("/", func(c *gin.Context) {
+		c.HTML(http.StatusOK, "index.tmpl", gin.H{
+			"title": "Main website",
```
Comment (BradHacker) [inline command-line-local-shortener/main.go]: Shouldn't need a title here.... this is what the `<title>....</title>` tag is for in the `html` file

File: command-line-local-shortener/main.go [hunk 5]
```diff
@@ -107,10 +107,12 @@ func main() {
 
 	router := gin.Default()
 
+	router.LoadHTMLGlob("templates/*")
 	router.GET("/:id", getReq)
-
-	router.GET("/home", func(c *gin.Context) {
-		c.JSON(200, gin.H{"hello": "world"})
+	router.GET("/", func(c *gin.Context) {
+		c.HTML(http.StatusOK, "index.tmpl", gin.H{
```
Comment (BradHacker) [inline command-line-local-shortener/main.go]: Update this file name to fit my other comment. (ie. `.tmpl -> .html`)

**Review by BradHacker — CHANGES_REQUESTED** [review]
Not entirely sure why the html file is called `index.tmpl`. Change that to `index.html` please.

Still a ways to go, but hey, it's a decent enough start.

File: command-line-local-shortener/main.go (hunk 4 above)
Comment (MiTo0o) [reply command-line-local-shortener/main.go]: can i just leave this blank then?

Comment (BradHacker) [reply command-line-local-shortener/main.go]: Yes, that should be fine

**Review by BradHacker — APPROVED** [review]

**Pull request merged**